### ISSUE TITLE
List repository branches

### ISSUE THREAD

**Issue by dustin** [dustin/py-github#27]
http://develop.github.com/p/repo.html

**Comment by dustin** [comment]
List branches within a repository.

Closed by 05b94a4a9c1e71d96eb2b3a640fc7d5e35a66750

**Comment by ghost** [comment]
Unfortunately this doesn't work with branches imported from git-svn, as / is used as tree path separator there. E.g. see http://github.com/api/v2/xml/repos/show/django/django/branches , which results in

```
xml.parsers.expat.ExpatError: not well-formed (invalid token)
```

when

```
gh.repos.branches('django', 'django')
```

is run.

The only way out would be to use JSON instead of XML, e.g. http://github.com/api/v2/json/repos/show/django/django/branches is of course well-formed JSON-wise.
